### DISCUSSION TITLE
Smart Meter - send status when arrives

### DISCUSSION THREAD

**Discussion by lzimmermann** [arendst/Tasmota#21457] [Q&A] [answered]
Hi all,
I have Tasmota running for an MT691 smart meter configured to send data through MQTT. The configuration is only done by a script.
Is there any way to set the sending of values to immediate? Configuring the time_period, I get data only every 10 seconds.
I don't see a parameter in the script that could change this.
Is there a Tasmota build that would send data as it arrives?

**Comment by sfromis** [comment, marked as answer]
Search the smart meter docs page for the word "immediately", and you should find something of interest.

**Reply by lzimmermann** [reply]
Thanks sfromis, think I read the docs several time but missed that one

**Reply by HofeBY2** [reply]
Take you Handycam and film the optical Eye of your Meter and you will notice, that he is sending in Blocks of aproximatly 3sec.

**Reply by lzimmermann** [reply]
nono, i get values more than once a second:
16:27:39.300	info	javascript.0 
16:27:39.314	info	javascript.0 
16:27:40.952	info	javascript.0 
16:27:41.780	info	javascript.0 
16:27:42.231	info	javascript.0 
16:27:44.235	info	javascript.0 
16:27:44.271	info	javascript.0 
16:27:45.016	info	javascript.0 
16:27:45.295	info	javascript.0 
16:27:45.920	info	javascript.0 
16:27:46.925	info	javascript.0 
16:27:47.925	info	javascript.0 
16:27:48.944	info	javascript.0 
16:27:49.923	info	javascript.0 
16:27:50.924	info	javascript.0 
16:27:52.106	info	javascript.0 
16:27:53.215	info	javascript.0 
16:27:53.926	info	javascript.0 
16:27:54.922	info	javascript.0 
16:27:55.776	info	javascript.0 
16:27:55.925	info	javascript.0 
16:27:56.926	info	javascript.0 
16:27:57.967	info	javascript.0

**Reply by sfromis** [reply]
Didn't you ask for **immediately** ? :stuck_out_tongue_closed_eyes:

**Reply by lzimmermann** [reply]
Absoloutly what I expected :-)
